index.js

```diff
@@ -19,10 +19,10 @@ module.exports = {
 		function build( data ) {
 			data.parts.push(
 				'<head>' +
-					'<script src="' + path.join( '/plugins', sinonPath ) + '"></script>' +
+					'<script src="/plugins/' + sinonPath.split( path.sep ).join( '/' ) + '"></script>' +
```

Comment:
Both UNIX and Windows path starts with ```/``` AFAICS. And this code adds preceding ```/``` for both platforms.